docs/release-notes.txt

```diff
@@ -6,6 +6,49 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.3
+^^^^^^^^^^^^^^
+
+**Release Date:** April 27, 2020
+
+- Add a tutorial for the new (experimental) Native API.
+
+- Add support for locally testing experiments via ``det e create --local``.
+
+- Add :class:`determined.experimental.Determined` class for accessing Experiment, Trial, and Checkpoint objects.
```

Comment:
Can we add links to the appropriate documentation (if it exists for the second and third point, at least)?

Reply:
sure